mz.mk

```diff
@@ -0,0 +1,287 @@
+##
```

Comment:
This file is IDE as well as specific to your local environment. Please remove it from the changeset